crates/core/src/db/datastore/system_tables.rs

```diff
@@ -237,229 +243,271 @@ st_fields_enum!(enum StVarFields {
 st_fields_enum!(enum StScheduledFields {
     "table_id", TableId = 0,
     "reducer_name", ReducerName = 1,
+    "schedule_name", ScheduleName = 2,
+    "at_column", AtColumn = 3,
+    "id_column", IdColumn = 4,
 });
 
-/// System Table [ST_TABLE_NAME]
-///
-/// | table_id | table_name  | table_type | table_access |
-/// |----------|-------------|----------- |------------- |
-/// | 4        | "customers" | "user"     | "public"     |
-fn st_table_schema() -> TableSchema {
-    #[allow(deprecated)]
-    TableSchema::from_def(
-        ST_TABLE_ID,
-        RawTableDefV8::new(
-            ST_TABLE_NAME.into(),
-            vec![
-                RawColumnDefV8::sys(StTableFields::TableId.name(), TableId::get_type()),
-                RawColumnDefV8::sys(StTableFields::TableName.name(), AlgebraicType::String),
-                RawColumnDefV8::sys(StTableFields::TableType.name(), AlgebraicType::String),
-                RawColumnDefV8::sys(StTableFields::TablesAccess.name(), AlgebraicType::String),
-            ],
+/// See the comment on [`SYSTEM_MODULE_DEF`].
+fn system_module_def() -> ModuleDef {
+    let mut builder = RawModuleDefV9Builder::new();
+
+    // System Table [ST_TABLE_NAME]
+    //
+    // | table_id | table_name  | table_type | table_access |
+    // |----------|-------------|----------- |------------- |
+    // | 4        | "customers" | "user"     | "public"     |
+    builder
```

Comment:
Note: I am going to change these to use `add_type`, since `St(...)Row` now implements `SpacetimeType`. I'll add asserts in this method that the fields are in the correct position and have the right labels.